### PR TITLE
Updated for OTP v2

### PR DESCRIPTION
**Pull request by ikespand**
- In Dockerfile, changed the maven link and also the base image
- Updated ReadMe with new arguments and tutorial
- Updated Docker-compose

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
-FROM java:8-alpine
-LABEL maintainer="Stepan Kuzmin <to.stepan.kuzmin@gmail.com>"
+FROM lpicanco/java11-alpine:11.0
```
Comment (stepankuzmin) [inline Dockerfile:1]: What is your reason to change the baseimage?

File: README.md [hunk 2]
```diff
@@ -4,49 +4,52 @@
...
 
-Build graphs using GTFS and OSM extract in the current directory:
+Build the docker image from the Dockerfile:
 
 ```shell
-docker run \
-  -v $PWD:/graphs \
-  -e JAVA_OPTIONS=-Xmx4G \
-  urbica/otp --build /graphs
+docker build -t my_docker_id/otp .
```
Comment (stepankuzmin) [inline README.md:10]: Why do you need this step? There is ready to use `urbica/otp` image

File: README.md [hunk 3]
```diff
@@ -56,30 +59,41 @@ mv portland.pbf ./graphs/portland
 
 ### Start up OTP
 
-Build graph
+Build the docker image from the Dockerfile:
+
+```shell
+docker build -t ikespand/otp .
```
Comment (stepankuzmin) [inline README.md:65]: What is the reason changing `urbica/otp` to `ikespand/otp`?

File: README.md [hunk 4]
```diff
@@ -4,49 +4,52 @@
...
+Run OTP server by loading the graph, and exposing OTP's port 8080 to our machine's port 8080:
 
 ```shell
 docker run \
   -p 8080:8080 \
-  -v $PWD:/graphs \
+  -v $PWD/graphs:/var/otp/graphs \
   -e JAVA_OPTIONS=-Xmx4G \
-  urbica/otp --basePath /data --server --analyst --autoScan --verbose
+  my_docker_id/otp --load /var/otp/graphs/city_name/
```
Comment (stepankuzmin) [inline README.md:30]: Same here

File: README.md [hunk 5]
```diff
@@ -4,49 +4,52 @@
...
 
 ```shell
 docker run \
-  -p 8080:8080 \
-  -v $PWD:/var/otp/graphs \
-  -e JAVA_OPTIONS=-Xmx4G \
-  otp --server --autoScan --verbose
+-v $PWD/graphs:/var/otp/graphs \
+-e JAVA_OPTIONS=-Xmx4G \
+my_docker_id/otp --build \
```
Comment (stepankuzmin) [inline README.md:19]: One can use `urbica/otp` image here

File: README.md [hunk 6]
```diff
@@ -56,30 +59,41 @@ mv portland.pbf ./graphs/portland
...
-Run OTP server:
+Run OTP server by loading the graph, and exposing OTP's port 8080 to our machine's port 8080:
 
 ```shell
 docker run \
   -p 8080:8080 \
   -v $PWD/graphs:/var/otp/graphs \
   -e JAVA_OPTIONS=-Xmx4G \
-  urbica/otp --server --autoScan --verbose
+  ikespand/otp --load /var/otp/graphs/portland
```
Comment (stepankuzmin) [inline README.md:85]: Saeme question here

File: README.md [hunk 7]
```diff
@@ -56,30 +59,41 @@ mv portland.pbf ./graphs/portland
...
+Build graphs using GTFS and OSM extract in the current directory:
 
 ```shell
 docker run \
-  -v $PWD/graphs:/var/otp/graphs \
-  -e JAVA_OPTIONS=-Xmx4G \
-  urbica/otp --build /var/otp/graphs/portland
+ -v $PWD/graphs:/var/otp/graphs \
+ -e JAVA_OPTIONS=-Xmx4G \
+ ikespand/otp --build \
```
Comment (stepankuzmin) [inline README.md:74]: Saeme question here

**Review by stepankuzmin — COMMENTED** [review]
Hey @ikespand

Thanks for the PR, but can you please explain your reasoning behind some changes in comments?

**Comment by tarbaig** [comment]
If I would Adresse the raised questions could this Mr be merged? We need an top 2 image and contributing seems reasonable.

**Comment by ikespand** [comment]
@stepankuzmin : Sorry, somehow I missed all updates related to this repo. This PR can be deleted now because @tarbaig have created a [new PR](https://github.com/urbica/docker-otp/pull/4) which answers all the quries.

**Pull request closed without merging**